index.js

```diff
@@ -385,6 +384,10 @@ Router.prototype.find = function find (method, path, version) {
       if (node === null) {
         return getWildcardNode(wildcardNode, method, originalPath, pathLenWildcard)
       }
+
+      const tempPath = originalPath.slice(0, originalPath.length - path.length)
+      const previousPath = `${tempPath.slice(tempPath.lastIndexOf('/') + 1, tempPath.length)}${path}`
```

Comment:
Can you use `var` instead of `const`?
```suggestion
      var previousPath = `${tempPath.slice(tempPath.lastIndexOf('/') + 1, tempPath.length)}${path}`
```
Same in the line above.